lib/mcb/render.rb

```diff
@@ -1,121 +1,220 @@
 module MCB
   module Render
-    class << self
-      def campuses_table(campuses, name: 'Campuses')
-        if campuses.all? { |campuse| campuse.respond_to? :keys }
-          campuses = hashes_to_ostructs campuses
+    def contacts_table(contacts, name: 'Contacts')
+      return if contacts.nil?
+
+      [
+        "#{name}:",
+        render_table_or_none(contacts, contacts_table_columns)
+      ]
+    end
+
+    # rubocop:disable Style/SymbolArray
```

Comment:
Ya, I dunno. I originally thought that we'd leave them as multi-line lists because any one of these columns might have more added to it ... formatters, extractors, etc. Now ... I'm not sure I care so much. I guess I'll change them.

Reply:
I don't really mind either way. Happy to leave as is, was just wondering about why.

Reply:
I think I was having a mini rebellion against rubocop's authoritarianism. :)